src/Tools/RepoUtil/RepoData.json

```diff
@@ -34,6 +34,7 @@
         "^src\\\\Samples.*",
         "^src\\\\Dependencies.*",
         ".*NuSpec.loc.*",
-        ".*Templates.*"
+        ".*Templates.*",
+        "^build\\\\MSBuildToolset.*"
```

Comment:
@jaredpar The build was failing because of mismatched references. I'll talk with you about this offline.